<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<routes id="context-b1dbae97-0793-4e4c-965d-729b3fa195be" xmlns="http://camel.apache.org/schema/spring">
    <route customId="false" id="route1">
        <from uri="file://data?fileName=book.xml&amp;renameUsingCopy=true"/>
        <doTry customId="true" id="doTry1">
            <to uri="validator:file:data/book.xsd" id="to1"/>
            <doCatch id="doCatch1">
                <exception>org.apache.camel.processor.validation.SchemaValidationException</exception>
                <log message="We have a problem!! ${exception.message}" id="log1"/>
                <to uri="file://garbage" id="to2"/>
            </doCatch>
        </doTry>
    </route>
</routes>
